test/Microsoft.AspNet.Mvc.FunctionalTests/FormatFilterTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.TestHost;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public class FormatFilterTest
+    {
+        private readonly IServiceProvider _services = TestHelper.CreateServices("FiltersWebSite");
+        private readonly Action<IApplicationBuilder> _app = new FiltersWebSite.Startup().Configure;
+
+        [Fact]
+        public async Task FormatFilter_NoExtensionInRequest()
```

Comment:
nit: In general it is easier to read and understand tests which follow the general pattern.```<Feature>_<WhatConditionIsBeingTested>_<ExpectedResult>```
In this case since we are already have FormatFilterTest as the class name we should probably rename methods like so
```FormatFilter_NoExtensionInRequest_IsNoOp``` or ```FormatFilter_NoExtensionInRequest_IsSuccessful```